### DISCUSSION TITLE
run rtsp server with a jetson nano cis camera via gstreamer

### DISCUSSION THREAD

**Discussion by Bark33p3r** [bluenviron/mediamtx#1169] [Questions]
Hello everyone,

I am unfortunately not that much into programming and need some help.

I have a Jetson Nano with a Mipi CIS camera running only with "gst-launch-1.0 nvarguscamerasrc sensor_id=0 ! nvoverlaysink".
The project is from here: https://github.com/JetsonHacksNano/CSI-Camera
It is possible to open a window with the "simple_cam.py" file.
I would like to publish the camera via rtsp.

Can anyone give me a hint?

Thanks a lot

**Comment by aler9** [comment]
```
gst-launch-1.0 nvarguscamerasrc ! 'video/x-raw(memory:NVMM),width=3840,height=2160,format=NV12,framerate=30/1' ! nvv4l2h265enc bitrate=3000000 ! rtspclientsink location=rtsp://localhost:8554/mycamera
```

**Reply by duong0411** [reply]
> ```
> gst-launch-1.0 nvarguscamerasrc ! 'video/x-raw(memory:NVMM),width=3840,height=2160,format=NV12,framerate=30/1' ! nvv4l2h265enc bitrate=3000000 ! rtspclientsink location=rtsp://localhost:8554/mycamera
> ```

WARNING: erroneous pipeline: no element "rtspclientsink"

**Reply by aler9** [reply]
This is not related to MediaMTX but is a GStreamer question. Make sure that `gstreamer1.0-rtsp` is installed.
